.github/workflows/validate.yml

```diff
@@ -57,7 +57,4 @@ jobs:
         run: go test -v -race -tags="fastlyinternaldebug nofastlyhostcalls" ./...
 
       - name: Tests - TinyGo
-        env:
-          GOARCH: wasm
-          GOOS: wasip1
-        run: tinygo test -v -tags="fastlyinternaldebug nofastlyhostcalls" ./...
+        run: tinygo test -v -target=wasip1 -tags="fastlyinternaldebug nofastlyhostcalls" ./...
```

Comment:
Personally, I'd love to reduce noise in the test output by dropping `-v` here and elsewhere. Maybe a different PR?

Reply:
Done.